### PR TITLE
Fix test and size in suggests

### PR DESCRIPTION
**Pull request by quaxsze**
Fix tags, mime and format suggest size.

### CODE REVIEW

File: udata/tests/api/test_tags_api.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def test_suggest_tags_api(self, api):
 
         for suggestion in response.json:
             assert 'text' in suggestion
-            assert suggestion['text'].startswith('test')
+            assert 'tes' in suggestion['text']
```
Comment (maudetes) [inline udata/tests/api/test_tags_api.py:34]: `startswith` is used in other suggest test. The same issue could happen there as well?
Ex: test_suggest_datasets_api, test_suggest_organizations_api...

File: udata/core/tags/api.py [hunk 2]
```diff
@@ -24,5 +24,5 @@ def get(self):
         '''Suggest tags'''
         args = parser.parse_args()
         q = normalize(args['q'])
```
Comment (maudetes) [inline udata/core/tags/api.py:26]: I don't think we should normalize tags in tag suggest. It returns unrelevant results by changing the query from `'co'` to `''`.

**Review by maudetes — COMMENTED** [review]
Thank you for this fix!

File: udata/core/tags/api.py (hunk 2 above)
Comment (quaxsze) [reply udata/core/tags/api.py:26]: This seems to be a wanted behaviour as the function returns a value of `''` if the query is inferior to `current_app.config['TAG_MIN_LENGTH']`
```
if len(value) < MIN_TAG_LENGTH:
    value = ''
```

Comment (maudetes) [reply udata/core/tags/api.py:26]: Agreed, but I don't think we want this normalization in the case of suggest.

**Review by maudetes — APPROVED** [review]
Thank you!

**Pull request merged**